### ISSUE TITLE
Math with paragraph break broken in latex

### ISSUE THREAD

**Issue by jnothman** [sphinx-doc/sphinx#2646]
At master, the following

``` rst
Its dual is

.. math::

   \min_{\alpha} \frac{1}{2} \alpha^T Q \alpha - e^T \alpha


   \textrm {subject to } & y^T \alpha = 0\\
   & 0 \leq \alpha_i \leq C, i=1, ..., n
```

generates

``` latex
Its dual is
\begin{align*}\!\begin{aligned}
\min_{\alpha} \frac{1}{2} \alpha^T Q \alpha - e^T \alpha\\

\textrm {subject to } & y^T \alpha = 0\\
& 0 \leq \alpha_i \leq C, i=1, ..., n\\
\end{aligned}\end{align*}
```

which breaks with

```
Runaway argument?
\!\begin {aligned} \min _{\alpha } \frac {1}{2} \alpha ^T Q \alpha - \ETC.
! Paragraph ended before \align* was complete.
<to be read again>
                   \par
l.12378
```

This was previously (e.g. sphinx==1.3.1)

```
Its dual is
\begin{gather}
\begin{split}\min_{\alpha} \frac{1}{2} \alpha^T Q \alpha - e^T \alpha\end{split}\notag\\\begin{split}
\textrm {subject to } & y^T \alpha = 0\\
& 0 \leq \alpha_i \leq C, i=1, ..., n\end{split}\notag
\end{gather}
```

**Comment by kiwifb** [comment]
Just joining the thread since I have bumped on it in #777.

**Comment by tk0miya** [comment]
It comes from the math directive contains two empty lines. Until 1.4.4, it did not accept multiple empty lines.
I fixed it at b3e82fa.
As a workaround, please make these empty lines single.

Thank you for reporting.

**Comment by kiwifb** [comment]
Yeah, this time it builds. Thank you.
